libsolidity/analysis/ReferencesResolver.cpp

```diff
@@ -332,7 +332,7 @@ void ReferencesResolver::endVisit(VariableDeclaration const& _variable)
 			case Location::Memory: return "\"memory\"";
 			case Location::Storage: return "\"storage\"";
 			case Location::CallData: return "\"calldata\"";
-			case Location::Default: return "none";
+			case Location::Unspecified: return "none";
```

Comment:
What about changing `none` to `unspecified` also? Could fit better

Reply:
No I actually take that back. That's used for user feedback, so `none` is more precise.